docs/developer/running-saleor/s3.mdx

```diff
@@ -37,17 +37,23 @@ Custom domain will allow you to use your CloudFront distribution or the public d
 
 ## Cross-Origin Resource Sharing
 
-You need to configure your S3 bucket to allow cross-origin requests for some files to be properly served (SVG files, Javascript files, etc.). Follow the below instructions in your S3 Bucket’s permissions tab, under the [CORS section](https://cloud.google.com/storage/docs/xml-api/put-bucket-cors).
-
-```xml
-<?xml version="1.0" encoding="UTF-8"?>
-<CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
-  <CORSRule>
-    <AllowedOrigin>*</AllowedOrigin>
-    <AllowedMethod>GET</AllowedMethod>
-    <AllowedMethod>HEAD</AllowedMethod>
-    <MaxAgeSeconds>3000</MaxAgeSeconds>
-    <AllowedHeader>*</AllowedHeader>
-  </CORSRule>
-</CORSConfiguration>
+You need to configure your S3 bucket to allow cross-origin requests for some files to be properly served (SVG files, Javascript files, etc.). Paste the code below in your S3 Bucket’s permissions tab, under the [CORS section](https://docs.aws.amazon.com/AmazonS3/latest/dev/cors.html).
```

Comment:
I would say "For example, a correct syntax can be the one attached below. Refer to the [CORS documentation of AWS](https://docs.aws.amazon.com/AmazonS3/latest/dev/cors.html)"

We do not recommend putting `*` in all cases, only for development it can be acceptable.

Reply:
Done